generatorv2/README.md

```diff
@@ -0,0 +1,134 @@
+
+Here are some examples of how you should **NOT** work with the toml file.
+
+Make sure that the UUID is the only thing inside of `[UUID]`, if there is, for example, an extra space so would that break it.
+Here is an example
+
+```toml
+# This would break it since it is an extra space between UUID and `]`
+[1e22cceb-c5e4-4562-9afe-aef07ad1eaf4 ]
+# This would break it since it is an extra space between UUID and `[`
+[ 1e22cceb-c5e4-4562-9afe-aef07ad1eaf4]
+```
+
+The script won't care if you write `include = true` since if it sees the UUID it will always take it as long as `include = false` is not written.
+The script will not work if anything is misspelled, although the part which gets `include = false` doesn't care if it gets an extra space or not.
+
+**NOTE:**
+You are also **NOT** allowed to write `include = false` more than once after each UUID.
+Since that can lead to errors in the generator.
+
+Bad way:
+
+```toml
+[1e22cceb-c5e4-4562-9afe-aef07ad1eaf4]
+description = "basic"
+include = false
+include = false
+```
+
+Good way:
+
+```toml
+[1e22cceb-c5e4-4562-9afe-aef07ad1eaf4]
+description = "basic"
+include = false
+```
+
+### Template
+
+The generator uses a template file to generate the test file.
+The template is located under the `.meta` for each exercise.
+
+This template has to be manually written for each exercise.
+The goal although is to make it so that you only have to write the template once and then it will be able to be used to generate new tests.
+
+The template file is written in [Embedded Ruby(ERB)][erb].
+ERB enables you to write Ruby code inside of the template file.
+It also means that the templates can be highly customizable since you can write any Ruby code you want.
+
+When writing the template file, it is recommended to look at already existing template files to get a better understanding of how it works.
+The template is getting a slightly modified version of the canonical data, so you can check out the [canonical data][canonical data] to see the data structure.
+The modification is that the cases which are not included in the toml file will be removed from the data structure.
+
+When writing the template so is it a special tool that can help with giving `# skip` and `skip` tags for tests.
+You simply have to call the `status` method.
+It will return either `# skip` or `skip` depending on if it is the first test case or not.
+
+Here is an example:
+
+```
+<%= status()%>
+<%= status()%>
+<%= status()%>
+```
+
+result:
+
+```
+# skip
+skip
+skip
+```
+
+### The Test Generator
+
+If all the earlier steps are done so can you run the generator.
+To run the generator you need to have a working Ruby installation and installed all gems in the Gemfile.
+The generator is located in the `bin` directory and is called `generator.rb`.
+
+To run the generator so do you have to be in the root directory and run the following command:
+
+```shell
+bundle exec ./bin/generate -e <exercise_slug>
```

Comment:
Either the line commented prior to this comment is incorrect, or this line is incorrect.